extensions/cornerstone/src/initMeasurementService.js

```diff
@@ -121,11 +121,35 @@ const connectToolsToMeasurementService = (
       const { toolName } = metadata;
 
       annotationModifiedEventDetail.uid = annotationUID;
-      annotationToMeasurement(toolName, annotationModifiedEventDetail);
+      annotationToMeasurement(toolName, annotationModifiedEventDetail, true);
```

Comment:
what is true? can you comment on it above it?

Reply:
will do